sdks/python/apache_beam/runners/worker/sdk_worker.py

```diff
@@ -362,6 +384,9 @@ def maybe_profile(self, instruction_id):
     else:
       yield
 
+  def has_active_processors(self):
```

Comment:
This isn't correct, as bundle_processor_cache is shared. Hopefully we don't need it though.